portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -74,52 +85,53 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String nickname = (String) entity.getProperty("nickname");
       String email = (String) userService.getCurrentUser().getEmail();
       ArrayList<String> messageReplies = (ArrayList) entity.getProperty("replies");
-      BlogMessage message = new BlogMessage(messageId, tag, comment, nickname, email, messageReplies, timestamp);
+      BlogMessage message = new BlogMessage(
+            messageId, tag, comment, nickname, email, messageReplies, timestamp);
       messages.add(message);
     }
-      
+    return BlogMessages;
+  }
+    
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get BlogMessages from Datastore.
+    List<BlogMessage> BlogMessages = getBlogsFromDatastore();
+
     // Create BlogHashMap Object and put BlogMessages in the map.
     BlogHashMap blogMap = new BlogHashMap();
-    blogMap.putInMap(messages);
+    blogMap.putInMap(BlogMessages);
 
     // Load messages from BlogHashMap and respond with gson.
-    LinkedList<BlogMessage> loadedBlogMessages = blogMap.getMessages(tagsToSearch, numberOfCommentsToDisplay);
-    
+    LinkedList<BlogMessage> loadedBlogMessages = blogMap.getMessages(
+        tagsToSearch, numberOfCommentsToDisplay);
+
     Gson gson = new Gson();
     response.setContentType("application/json;");
       
     response.getWriter().println(gson.toJson(loadedBlogMessages));
     return;
-    }
+  }
     
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get Post parameters.
-    String message = request.getParameter("text-input");
-    String nickname = request.getParameter("sender");
+    final String MESSAGE = request.getParameter("text-input");
```

Comment:
When I said line 118&119 should be constants, I actually meant the parameter "text-input" & "sender". The variable themselves can be just local variables like before.